app/src/main/AndroidManifest.xml

```diff
@@ -270,6 +270,7 @@
             android:name=".feature.marketing.MarketingActivity"
             android:theme="@style/Hedvig.Theme.Marketing"
             android:windowSoftInputMode="adjustResize" />
+        <activity android:name=".feature.claims.ui.ClaimsFlowActivity" />
```

Comment:
Do we want this to have a Theme too? Do we rely 100% on everything coming from Odyssey?
Will this mean that if there's anything that is no defined 1:1 equivalent with the local theme, things inside the odyssey world may look off?

Reply:
We have set a theme for `<application>`, that should be enough right?

Reply:
Yep totally, my bad, read up on this a bit. This means that on the app manifest in 18 places where we define `android:theme="@style/Hedvig.Theme"` those could be removed too!